lib/ansible/modules/cloud/rackspace/rax_cbs_attachments.py

```diff
@@ -32,7 +32,7 @@
     description:
       - The device path to attach the volume to, e.g. /dev/xvde
     default: null
-    required: true
+    required: false
```

Comment:
Add note to the description that this changes in v2.3, it was a required parameter before that.